spec.html

```diff
@@ -278,6 +278,12 @@ <h3>Non-terminals</h3>
                     <td class="fix ex">"\x05"</td>
                     <td>MD5</td>
                   </tr>
+                  <tr>
+                    <td class="fix nt"></td>
+                    <td class="fix op">|</td>
+                    <td class="fix ex">"\x06"</td>
+                    <td>Client Side Encryption <a href="https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/subtype6.rst">ciphertext</a></td>
```

Comment:
This seems like the wrong thing to link to.  The spec is large and doesn't actually address the matter at hand here, which IMO is the structure of the binary value contained.

Maybe better to call this `Encrypted BSON value` and just say that the format is unspecified, since we don't expect third-party applications to be able to parse the value in any meaningful way

Reply:
That is the subtype 6 spec, not the full FLE spec. I think it's possible (though unlikely) that a third-party application could encrypt/decrypt a BSON subtype 6 given the description in that spec.

Also including the link seems in line with how Decimal128 and ObjectID link to specs of their binary layouts.

Reply:
I should have clicked the link... sorry, I forgot we actually had a separate spec for this!

I still prefer the name to be `Encrypted BSON value` rather than `Client side encryption`

Reply:
Done.